### PR TITLE
update README to reflect cronjob changes

### PR DESCRIPTION
**Pull request by digorgonzola**

### CODE REVIEW

**Review by akashisama — CHANGES_REQUESTED** [review]

**Comment by akashisama** [comment]
Will we need to be careful about the promotion order of this vs the changes in chef (https://github.com/aodn/chef/pull/1182/files)?

File: README.md [hunk 1]
```diff
@@ -13,19 +13,29 @@ FAIMMS/faimms_data_rss_channels_process
...
+|:--|:--|:--|
+| $ARCHIVE_DIR | /mnt/ebs/archive | Archive |
+| $ARCHIVE_IMOS_DIR | /mnt/ebs/archive | Archive |
+| $INCOMING_DIR | /mnt/ebs/incoming |Incoming |
+| $ERROR_DIR | /mnt/ebs/error |Errors |
+| $WIP_DIR | /mnt/ebs/wip |Work In Progress tmp dir |
+| $DATA_SERVICES_DIR | /mnt/ebs/data-services | Where this git repo is deployed |
+| $DATA_SERVICES_TMP_DIR | nil | |
+| $EMAIL_ALIASES | /etc/incoming-aliases | List of configured aliases |
+| $PYTHONPATH | $DATA_SERVICES_DIR/lib/python | Location of data-services python interpreter |
```
Comment (mhidas) [inline README.md:32]: There is no python interpreter there - AFAIK these scripts usually run on the system python!
```suggestion
| $PYTHONPATH | $DATA_SERVICES_DIR/lib/python | Location of data-services python scripts/modules |
```

File: README.md [hunk 2]
```diff
@@ -13,19 +13,29 @@ FAIMMS/faimms_data_rss_channels_process
...
-|$LOG_DIR          |/mnt/ebs/log/data-services |Designated log dir             |
+|Name |Default | Purpose|
+|:--|:--|:--|
+| $ARCHIVE_DIR | /mnt/ebs/archive | Archive |
+| $ARCHIVE_IMOS_DIR | /mnt/ebs/archive | Archive |
+| $INCOMING_DIR | /mnt/ebs/incoming |Incoming |
+| $ERROR_DIR | /mnt/ebs/error |Errors |
+| $WIP_DIR | /mnt/ebs/wip |Work In Progress tmp dir |
+| $DATA_SERVICES_DIR | /mnt/ebs/data-services | Where this git repo is deployed |
+| $DATA_SERVICES_TMP_DIR | nil | |
```
Comment (mhidas) [inline README.md:30]:
```suggestion
| $DATA_SERVICES_TMP_DIR | /mnt/ebs/tmp | Temp dir for data services work (not on root partition like /tmp) |
```

File: README.md [hunk 3]
```diff
@@ -13,19 +13,29 @@ FAIMMS/faimms_data_rss_channels_process
...
-|$INCOMING_DIR     |/mnt/ebs/incoming          |Incoming                       |
-|$WIP_DIR          |/mnt/ebs/wip               |Work In Progress tmp dir       |
-|$DATA_SERVICES_DIR|/mnt/ebs/data-services     |Where this git repo is deployed|
-|$LOG_DIR          |/mnt/ebs/log/data-services |Designated log dir             |
+|Name |Default | Purpose|
+|:--|:--|:--|
+| $ARCHIVE_DIR | /mnt/ebs/archive | Archive |
+| $ARCHIVE_IMOS_DIR | /mnt/ebs/archive | Archive |
+| $INCOMING_DIR | /mnt/ebs/incoming |Incoming |
+| $ERROR_DIR | /mnt/ebs/error |Errors |
```
Comment (mhidas) [inline README.md:27]:
```suggestion
| $ERROR_DIR | /mnt/ebs/error | Dir. to store incoming files that cause pipeline errors |
```

**Review by mhidas — COMMENTED** [review]
Just a few suggestions

**Review by akashisama — APPROVED** [review]

**Pull request merged**